services/docker/docker.service.js

```diff
@@ -80,7 +80,36 @@ module.exports = class Docker extends LegacyService {
         })
       })
     )
+  }
+}
```

Comment:
I split this up because new-style services can only be in one category, which means legacy service examples have to be split along category lines.